benchmarks/opmphm.c

```diff
@@ -1161,6 +1974,29 @@ static size_t getPower (size_t p, size_t q)
 	return result;
 }
 
+/**
+ * @brief Compares two integer.
```

Comment:
description misleading, better to write "comparison between integers suitable as qsort callback"